src/app/shared/components/disk-offering/disk-offering-dialog/disk-offering-dialog.component.ts

```diff
@@ -47,4 +53,10 @@ export class DiskOfferingDialogComponent {
     const isNoDiskOfferings = !this.diskOfferings.data.length;
     return isDiskOfferingNotSelected || isNoDiskOfferings;
   }
+
+  private checkResourcesLimit() {
+    this.minSize = this.authService.getCustomDiskOfferingMinSize();
+    const diskSize = this.selectedDiskOffering.disksize === 0 ? this.minSize : this.selectedDiskOffering.disksize;
+    this.resourcesLimitExceeded = Number(this.account.primarystorageavailable) < diskSize;
```

Comment:
As I know CloudStack has a primary storage limit and secondary storage limit.
Are you sure that all created disk from the "Storage" section is a primary storage?

Reply:
I've checked it.